src/main/webapp/test/mentors.test.js

```diff
@@ -0,0 +1,42 @@
+import * as Mentors from '../mentors.js';
```

Comment:
```suggestion
import * as mentors from '../mentors.js';
```
https://google.github.io/styleguide/jsguide.html#es-module-imports section 3.4.1.3.1 (feel like such a nerd lol)

Reply:
lowerCamelCase making an appearance :)